testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/management/cli/DeploymentOverlayCLITestCase.java

```diff
@@ -210,7 +210,7 @@ public static void after() throws Exception {
         ear2.delete();
         FileUtils.deleteDirectory(ear2_exploded);
         replacedLibrary.delete();
-        replacedAjsp.delete();
+//        replacedAjsp.delete();
```

Comment:
Methods @BeforeClass and @AfterClass are invoke twice (at least on my local machine).
replacedAjsp - this file is in target location... other files are generated - it is reason why it works after comment.

I am trying to find out where is the problem.